src/tables/sortable-table.js

```diff
@@ -109,43 +155,39 @@ function SortableTable ({
 SortableTable.propTypes = propTypes
 SortableTable.defaultProps = defaultProps
 
-const WrappedTable = sortable()(SortableTable)
-
-// Passes relevant sortable props
-function SortableTableWrapper ({ initialColumn, children, disableSort, disableReverse, onChange, ...rest }) {
+// Passes relevant SortableTable props and applies valueGetters...
+function SortableTableWrapper({
```

Comment:
Now that we're not using a HOC, you can remove this wrapper and move the functionality into the main component.

Reply:
Done.